values.yaml

```diff
@@ -338,6 +338,10 @@ server:
   serviceAccount:
     annotations: {}
 
+  # Definition of the statefulSet used to run Vault.
+  statefulSet:
+    annotations: {}
```

Comment:
```suggestion
    # Extra annotations for the statefulSet. This can either be YAML or a
    # YAML-formatted multi-line templated string map of the annotations to apply
    # to the statefulSet.
    annotations: {}
```